ServiceStack/src/ServiceStack/Keywords.cs

```diff
@@ -70,6 +70,7 @@ public static class Keywords
     public const string TraceId = nameof(TraceId);
     public const string FileName = nameof(FileName);
     public const string ClaimsPrincipal = nameof(ClaimsPrincipal);
+    public const string ServiceGatewayResponseStatus = nameof(ServiceGatewayResponseStatus);
```

Comment:
We could reuse the ResponseStatus keyword potentially as I'm not sold ServiceGatewayResponseStatus is the best name. Open to other suggestions

Reply:
Yeah please change it to `ResponseStatus`

Reply:
Changed to ResponseStatus